core/CronArchive.php

```diff
@@ -1023,12 +1023,13 @@ public function canWeSkipInvalidatingBecauseThereIsAUsablePeriod(Parameters $par
             Date::now()->subSeconds(Rules::getPeriodArchiveTimeToLiveDefault($params->getPeriod()->getLabel()));
 
         // empty plugins param since we only check for an 'all' archive
-        list($idArchive, $visits, $visitsConverted, $ignore, $tsArchived) = ArchiveSelector::getArchiveIdAndVisits($params, $minArchiveProcessedTime, $includeInvalidated = $isPeriodIncludesToday);
+        $archiveInfo = ArchiveSelector::getArchiveIdAndVisits($params, $minArchiveProcessedTime, $includeInvalidated = $isPeriodIncludesToday);
+        $idArchive = $archiveInfo['idArchives'];
```

Comment:
```suggestion
        $idArchive = $archiveInfo['idArchives'];
        $tsArchived = $archiveInfo['tsArchived'];
```

and then below the line can stay as it was.